src/__tests__/qrzApiClient.test.ts

```
@@ -57,6 +57,45 @@ describe('QrzClient', () => {
     })
   })
 
+  describe('baseUrl', () => {
+    const originalWindow = global.window;
+    let consoleSpy: ReturnType<typeof vi.spyOn>;
+
+    beforeEach(() => {
+      delete (global as any).window;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)